src/controls/paginator/page-link.js

```diff
@@ -22,8 +22,9 @@ function PageLink({ className, active, onClick, children, ...rest }) {
   return (
     <li className={classnames(className, { active: active })}>
       <a
+        role="link"
```

Comment:
Allows us to keep these as links which is nice for having them show up in the screenreader "Links" menu